README.org

```diff
@@ -4,33 +4,38 @@ Very early stage Emacs integration for [[https://github.com/kRITZCREEK/psc-ide][
 
 [[./screenshot-1.png]]
 
-* Installation
-
-Add the following to your =~/.emacs=:
+* Usage
 
-#+BEGIN_SRC elisp
-;; specify the path to the plugin directory
-(add-to-list 'load-path "~/.emacs.d/lisp/psc-ide-emacs/")
+Add following into your ~/.emacs :
 
-;; specify path to the 'psc-ide' executable
-(customize-set-variable 'psc-ide-executable "/path/to/psc-ide")
+#+BEGIN_SRC Emacs Lisp
 
+(add-to-list 'load-path "~/.emacs.d/lisp/psc-ide-emacs/") ;; where your psc-ide.el file
 (require 'psc-ide)
 
 (add-hook 'purescript-mode-hook
-      (lambda ()
-          (add-to-list 'company-backends 'company-psc-ide-backend)
-          (company-mode)
-          (message "Purescript IDE enabled")
-      ))
-(define-key purescript-mode-map (kbd "C-<SPC>") 'company-complete)
-#+END_SRC
+          (lambda ()
+              (add-to-list 'company-backends 'company-psc-ide-backend)
+              (company-mode)
+              (psc-ide-mode)
+              (message "Purescript IDE enabled")
+          ))
 
-* Usage
+(setq psc-ide-executable "/path/to/psc-ide") 
+(setq psc-ide-server-executable "/path/to/psc-ide-server")
+
+#+END_SRC
 
-The first two steps will be automated in the future:
 
-- Start ~psc-ide-server~ in your purescript project dir.
-- Load the modules you want to use as completion sources: ~echo "load Prelude" | psc-ide~.
 - Visit a ~.purs~ buffer inside your PureScript project.
-- Put your cursor to the right of an identifier and do ~M-x company-complete~ or press Control-Space. 
+- Start the server with shortcut "C-c C-s"
+- Load module with shortcut "C-c C-l"
+
+Start hacking!
+
+Available keybindings:
```

Comment:
If you add a `*` before this, it turns into a separate heading, which I think might be nice.